proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/SamlResponseExceptionMapperResourceRuleTest.java

```diff
@@ -37,12 +37,14 @@
 
     @Before
     public void before() {
-        when(sessionStore.getSession(any(String.class))).thenReturn(
-                new GatewaySessionData("HubRequestId", "EidasRequestId", "EidasDestination",
-                        "EidasConnectorPublicKey", "EidasRelayState"));
-
-        when(samlFormViewBuilder.buildResponse("EidasDestination", (String) null, "Continue", "EidasRelayState"))
-                .thenReturn(new SamlFormView("postUrl", "samlMessageType", "encodedSamlMessage", "submitText"));
+        when(sessionStore.getSession(any(String.class))).thenReturn(new GatewaySessionData("HubRequestId","EidasRequestId","EidasDestination","EidasConnectorPublicKey","EidasRelayState"));
+        String nullString = null;
+        when(samlFormViewBuilder.buildResponse("EidasDestination", nullString, "EidasRelayState"))
```

Comment:
I assume this was done to shut up codacy or something, but I feel like it should have been a separate PR as it doesn't seem to have anything to do with this one?